### ISSUE TITLE
Modify CTAN upload format

### ISSUE THREAD

**Issue by AndyClifton** [AndyClifton/accessibility#29]
Feedback from CTAN, very heavily summarized:

> please remove the .tds.zip from the submission and include the `.sty` file in any later submissions.

**Comment by AndyClifton** [comment]
I upload the `accessibility.zip` file to CTAN. 

`accessibility.zip` contains
- accessibility.tds.zip
- accessibility/
  - accessibility.dtx, accessibility.ins
  - accessibility.pdf
  - accessibility-Anleitung-DE.pdf
  - CHANGELOG.md
  - README.md

This is generated by...
- `source/v2x/prepare_for_CTAN.sh`
- which calls `ctanify`, which is installed on my system.
- `ctanify` generates the .zip file.

So.. I need to modify the call to `ctanify`. Currently this is...

```console
ctanify --pkgname accessibility \
  accessibility.ins \
  accessibility.pdf \
  accessibility-Anleitung-DE.pdf \
  README.md \
  CHANGELOG.md \
 "accessibility-Anleitung-DE.pdf=doc/latex/accessibility" \
 "accessibility.pdf=doc/latex/accessibility"
```
How do I tell ctanify to not include the .tds.zip?

**Comment by AndyClifton** [comment]
Try...

``` console
ctanify --pkgname accessibility \
  accessibility.ins \
  accessibility.pdf \
  accessibility-Anleitung-DE.pdf \
  README.md \
  CHANGELOG.md \
  --no-tds
```

**Comment by AndyClifton** [comment]
Fixed in commit 09dde82.
